guides/basic-use/index.md

```diff
@@ -20,21 +22,28 @@ This will make the `ember` command available throughout your project folders. Th
 
 ### Additional steps for Mac and Linux users
 
-For Mac and Linux users we recommend installing [Watchman](https://facebook.github.io/watchman/). Watchman helps correct for some buggy and inefficient file watching behavior. Do not use the `npm` package by the same name.
+For Mac and Linux users, we recommend installing [Watchman](https://facebook.github.io/watchman/). Do not use the `npm` package by the same name.
+
+Why is Watchman needed?
```

Comment:
This formats as part of same paragraph as below, should be formatted separately with emphasis